### PR TITLE
Fix/lts feedback

### PR DESCRIPTION
**Pull request by Bluesmile82**
These fix some feedback changes proposed on:
https://onewri.sharepoint.com/:w:/s/climatewatch/EZ6y5u8bGhZHoSkXkXX2EtgBGlOVcUNRhAcDFLm8rv2jRQ?e=0fss7r

LTS explore page:

- Update column name in the table to "Long-term Strategies Document"
- Add LTS submission as a default for map dropdown selection.

NDCS explore page:

- Fix table column indicator which was not matching with the selected indicator on the map.
- Fix titleLinks sorting (cw-components update)

NDC search map
- Add zoom buttons and dots on islands

Overview page: 
- Little proposed text changes

NDC country page:
- Search styles fixing

DESIGN CHANGES:
https://www.pivotaltracker.com/story/show/170818752

- NDC country: On mobile Cards show all the content (without scroll)
- Fix mobile styles for NDC/LTS country pages
- Change gray on the NDC search map. Also in charts and icons. Remove unused icon
- Fix NDCS 2020 Tracker cards: Background white and box-shadow.

### CODE REVIEW

**Comment by kfilip94** [comment]
checking

**Comment by kfilip94** [comment]
Please unify styles of the zoom buttons in NDC Search, they look different from other pages with the map.
NDC Search:
Screenshot from 2020-01-22 10 32 53
And other places like Explore LTS, NDCS:
Screenshot from 2020-01-22 10 32 08

File: app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js [hunk 1]
```diff
@@ -42,6 +49,50 @@ export const getDefaultColumns = createSelector(
   }
 );
 
+export const tableGetSelectedData = createSelector(
+  [getIndicatorsParsed, getCountries],
+  (indicators, countries) => {
+    if (!indicators || !indicators.length || !indicators[0].locations) {
```
Comment (kfilip94) [inline app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js]: Please add `countries` guard to this selector:
```suggestion
    if (!indicators || !indicators.length || !indicators[0].locations || !countries) {
```

File: app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js [hunk 2]
```diff
@@ -42,6 +49,50 @@ export const getDefaultColumns = createSelector(
...
+        }
+      });
+      return row;
+    });
+  }
+);
+
+const addIndicatorColumn = createSelector(
+  [tableGetSelectedData, getMapIndicator, getSelectedIndicatorHeader],
+  (data, selectedIndicator, selectedIndicatorHeader) => {
```
Comment (kfilip94) [inline app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js]: Same here, please add guards for `selectedIndicator` `selectedIndicatorHeader` ;)

**Comment by kfilip94** [comment]
Hey @Bluesmile82, great job here! I left a few comments above ;)

**Comment by Bluesmile82** [comment]
Fixed and rebased! Thanks

**Pull request merged**